pkgs/tools/security/sudo/default.nix

```diff
@@ -14,11 +14,11 @@
 
 stdenv.mkDerivation rec {
```

Comment:
What about using the finalAttrs pattern?

Reply:
Sure, please send me a follow-up PR and I'll be happy to review it.